docs/api/README.md

```diff
@@ -539,6 +539,32 @@ function* takeOneAtMost() {
 }
 ```
 
+### `flush(channel)`
+
+Creates an effect that instructs the middleware to flush all buffered items from the channel. Flushed items are returned back to the saga, so can they be utilized.
```

Comment:
What do you mean? Just indicating that they can be utilized by the user if he wants to and thats it not mandatory?